domain/FlowdockConfiguration.ts

```diff
@@ -1,15 +0,0 @@
-class FlowdockConfiguration {
```

Comment:
Is it possible to support both platforms and just configure which you use?

Reply:
We cannot test on this platform in the future, so I decided to drop the support for it